### DISCUSSION TITLE
Confirmation before container rebuild

### DISCUSSION THREAD

**Discussion by windmillcode0** [devcontainers/community#93] [Ideas]
Greetings on the vscode command for rebuild container, this would delete all work in the container, there should be a confirmation message before rebuilding of the container

**Comment by eljog** [comment]
Thank you for using devcontainers and opening this discussion.

Typically it should only delete the container and not the mounted volume that hosts the code.
You may open  a feature request for the confirmation message at https://github.com/microsoft/vscode-remote-release/issues ?

**Comment by windmillcode0** [comment]
It went and deleted my code sadly, and there was no indication of a volumne anywhere, but my finger stuttered on the "Rebuild Container without cache option"

**Reply by eljog** [reply]
Thank you for sharing the additional context.
Please open an issue at https://github.com/microsoft/vscode-remote-release/issues so that the vscode extension team can help you
